src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRentalPrice = 40;
+  const discountForSevenDays = 50;
+  const discountForThreeDays = 20;
+  const DAYS_FOR_SMALL_DISCOUNT = 3;
+  const DAYS_FOR_LARGE_DISCOUNT = 7;
+  const rentalDays = days;
+  let totalCost = dailyRentalPrice * rentalDays;
+
+  return rentalDays >= DAYS_FOR_LARGE_DISCOUNT
+    ? (totalCost -= discountForSevenDays)
+    : rentalDays >= DAYS_FOR_SMALL_DISCOUNT
+      ? (totalCost -= discountForThreeDays)
+      : totalCost;
```

Comment:
It will be more readable
```suggestion
    if (rentalDays >= LONG_TERM) {
      return totalCost - DAYS_FOR_LARGE_DISCOUNT;
    }

    if (rentalDays >= MEDIUM_TERM) {
      return totalCost - DAYS_FOR_SMALL_DISCOUNT;
    }

    return totalCost;
```